src/components/addItems/AddItemButton.tsx

```diff
@@ -1,27 +1,39 @@
 import { useState } from "react";
 import { DataStore } from "aws-amplify";
+import { Button } from "@mui/material";
 import emailjs from "emailjs-com";
+import { GridRowsProp } from "@mui/x-data-grid";
 import { Category, Item, ItemCategory } from "../../models";
 import "../../styles/AddItemButton.css";
 
-function AddItemButton() {
+function AddItemButton({
+  isDisabled,
+  rows,
+  setRows,
+  setIsDrawerOpen,
+}: {
+  isDisabled: boolean;
```

Comment:
It is cleaner to create a new Typescript type that defines the proptypes and then setting it here. See Table.tsx